e2e/tests/Login/SuccessfullyLogIn.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from '@playwright/test';
+
+import {
+  getAccessToken,
+  resetDB,
+} from '../../../services/API-service/test/helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/');
+  await loginPage.login(
+    process.env.USERCONFIG_MALAWI_EMAIL,
+    process.env.USERCONFIG_MALAWI_PASSWORD,
+  );
+});
+
+test('Successfully Login', async ({ page }) => {
+  await page.waitForURL((url) => url.pathname.startsWith('/'));
```

Comment:
Is this assertion actually correct? When I get the test running and then  print out the 'url' variable here, it shows me that ur.pathname = '/login', which passes the assertion but is not actually what we want to test right? We want to test that it goes from the login page to the dashboard page.

Reply:
As discussed the test will assert that user was redirected to base URL.